libs/service-portal/documents/src/components/DocumentCard/DocumentCard.tsx

```diff
@@ -86,7 +115,8 @@ const DocumentCard: FC<Props> = ({ document }) => {
         key={document.id}
         loading={loading}
         cta={{
-          onClick: handleOnClick,
+          externalUrl: getEdgecaseDocument(document)?.url,
```

Comment:
Do we need a fallback in case this is empty? @andriandresson

Reply:
`onClick: handleOnClick` is never empty. ActionCard Component provides the logic to render either Link, if externalUrl prop is provided, or Button, and onClick prop is always provided (not optional prop).